test/src/test/java/org/corfudb/runtime/view/stream/OpaqueStreamTest.java

```diff
@@ -64,6 +70,44 @@ public void testMagicByte() {
 
     }
 
+    @Test
+    public void serDeSer() {
```

Comment:
Can we add a test description and I'm guessing the name was serializeDeserializeTest?

Reply:
I completely forgot about that test. Let me clean it up.